ndless-sdk/bin/nspire-tools

```diff
@@ -61,7 +81,17 @@ case "$1" in
 		cp "${DIRNAME}/main.c.tpl" main.c
 		;;
 	path)
-		echo "$(dirname "$DIRNAME")/"
+		echo "$NDLESS/"
+		;;
+	gcc)
```

Comment:
This seems odd, why only `gcc` and not other tools?

IMO you can remove this, as with `toolchainbin` it's redundant.